pkg/cri/sbserver/podsandbox/sandbox_stop.go

```diff
@@ -134,5 +135,5 @@ func (c *Controller) waitSandboxStop(ctx context.Context, sandbox sandboxstore.S
 // cleanupUnknownSandbox cleanup stopped sandbox in unknown state.
 func cleanupUnknownSandbox(ctx context.Context, id string, sandbox sandboxstore.Sandbox) error {
 	// Reuse handleSandboxExit to do the cleanup.
-	return handleSandboxExit(ctx, sandbox)
+	return handleSandboxExit(ctx, &eventtypes.TaskExit{ExitStatus: 0, ExitedAt: protobuf.ToTimestamp(time.Now())}, sandbox)
```

Comment:
Would you please to align with existing logic here? https://github.com/containerd/containerd/blob/main/pkg/cri/server/sandbox_stop.go#L193

For the unknown sandbox exit code, we use 255.